# HG changeset patch
# User John Smith <john@example.com>
# Date 1474010779 -28800
#      Fri Sep 16 15:26:19 2016 +0800
# Node ID 39d84331d692c8b0beb94da7cba0c5af83ee070d
# Parent  0000000000000000000000000000000000000000
First patch ever, I don't know what I'm doing!

diff --git a/foo b/foo
new file mode 100644
--- /dev/null
+++ b/foo
@@ -0,0 +1,1 @@
+bar
